### PR TITLE
volume_status: use py3.command_output

### PR DESCRIPTION
**Pull request by vvoland**
This PR partially relies on #1267 

volume_status module calls commands by using functions directly from subprocess
instead of using helper function from py3. It also fails to parse some commands
output with non-English locale set. 

This PR makes volume_status use py3.command_output function, which in
conjunction with #1267 fixes command parsing issue.

### CODE REVIEW

File: py3status/modules/volume_status.py [hunk 1]
```diff
@@ -157,10 +158,10 @@ def setup(self, parent):
...
         try:
-            perc = check_output(self.cmd + ["--get-volume"])
-        except CalledProcessError as cpe:
+            perc = self.command_output(self.cmd + ["--get-volume"])
+        except CommandError as ce:
             # pamixer throws error on zero percent. see #1135
-            perc = cpe.output
+            perc = ce.output
 
         perc = perc.decode().strip()
```
Comment (lasers) [inline py3status/modules/volume_status.py]: It breaks. We need to remove this `.decode()` too.

Comment (vvoland) [reply py3status/modules/volume_status.py]: Right, fixed.

**Comment by tobes** [comment]
Looks great.  I'll do some testing too when I get the chance.  @lasers thanks for your testing

**Comment by tobes** [comment]
@vvoland thanks for this cool refactoring - sorry it took so long to get in

**Pull request merged**